src/CellValue.ts

```diff
@@ -103,7 +105,15 @@ export class Exporter {
   }
 
   private detailedError(error: CellError): DetailedCellError {
-    return new DetailedCellError(error, this.config.translationPackage.getErrorTranslation(error.type))
+    let address = undefined
+    if(error.address !== undefined) {
+      if (error.address.sheet === NamedExpressions.SHEET_FOR_WORKBOOK_EXPRESSIONS) {
+        address = this.namedExpressions.namedExpressionInAddress(error.address.row)?.displayName
+      } else {
+        address = simpleCellAddressToString(this.sheetIndexMapping, error.address, -1)
+      }
+    }
+    return new DetailedCellError(error, this.config.translationPackage.getErrorTranslation(error.type), address)
```

Comment:
I had a mixed feeling about another import in CellValue, mixing parser and interpreter imports with CellValue. That the address mapping should be done during the value request aka exporting.

Then I realized this file consists mostly of classes needed by the exporter. Shouldn't we rename it or extract the exporter?

Reply:
:+1: